pkg/doc/did/doc.go

```diff
@@ -104,6 +104,7 @@ func Parse(did string) (*DID, error) {
 // Doc DID Document definition.
 type Doc struct {
 	Context              []string
+	Base                 string
```

Comment:
Does Base need to be exposed? @fqutishat @sandrask 

Instead suggest using a function that returns an absolute URI for a DID URI property.

Reply:
@troyronda no need to expose it

Reply:
done